File: TugasPraktikum04/Bahasa C/PRAK403-2310817320013-AikoAnatashaWendiono.c
#include<stdio.h>

int main() {
    int awal, akhir, angka1, angka2;

    printf("Input: ");
    scanf("%d %d", &angka1, &angka2);

    if (angka1 > angka2){
        for (awal = angka1, akhir = angka2; awal >= angka2, akhir <= angka1; awal--, akhir++ ){
            printf("%d %d ", awal, akhir);
            if (awal == angka2){
                break;
            } else {
                printf("- ");
            }
            
        }

    }
    else {
        for (awal = angka1, akhir = angka2; awal <= angka2, akhir >= angka1; awal++, akhir--){
            printf("%d %d ", awal, akhir);
            if (awal == angka2){
                break;
            }
            else {
                printf("- ");
            }
        }
    }

return 0;
}

File: TugasPraktikum04/Bahasa Python/PRAK403-2310817320013-AikoAnatashaWendiono.py
angka1, angka2 = map(int, input("Input: ").split())

if angka1 > angka2:
    awal, akhir = angka1, angka2
    awal_2, akhir_2 = -1, 1
else:
    awal, akhir = angka1, angka2
    awal_2, akhir_2 = 1, -1

while True:
    print(f"{awal} {akhir}", end=" ")
    if awal == angka2:
        break
    else:
        print("- ", end="")
    awal += awal_2
    akhir += akhir_2
